Makefile

```diff
@@ -127,6 +133,9 @@ image-gcloud: installer
 image-vanilla: installer
 	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out --privileged $(DOCKER_ARGS) autonomy/installer:$(TAG) image -l
 
+publish-aws:
+	@packer build -var "visibility=all" -var "regions=$(AWS_PUBLISH_REGIONS)" -var "region=us-west-2" -var "version=$(TAG)" hack/installer/packer.json
```

Comment:
At first, I tough you would deploy the changes for the AWS AMI building before the automatic deployment of the docker images. That's why I needed to launch the packer command without docker because with docker I wouldn't have an up-to-date packer.json configuration.
But then we talked about it and I forgot to make the changes. I'll certainly move that to docker.